project.Makefile

```diff
@@ -1 +1,5 @@
-## Add your own custom Makefile targets here
+CLI=$(RUN) python -m src.comp_loinc
+
+.PHONY:
+build:
+	$(CLI) generate-all
```

Comment:
## `make build`
FYI. Runs the `generate-all` command. I think this is correct for a general non-custom build.